java/java.mx.project/release/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,442 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What is the specific use of this file? Is it better to have it hardcoded, or read the version from mx?

Reply:
This file was added by @dbalek to support [Eclipse Formatter by Benno](http://plugins.netbeans.org/plugin/50877/eclipse-code-formatter-for-java-eclipse-mars-4-5).